### ISSUE TITLE
ds format in GREIT algorithm 

### ISSUE THREAD

**Issue by JeanRintoul** [liubenyuan/pyEIT#6]
I've noticed that running JAC and Back Projection algorithm result in ds format which can be overlayed on the mesh as shown below: 
![desired_figure](https://user-images.githubusercontent.com/315476/38650466-d8d5716c-3db0-11e8-8ad0-997ec9510955.png)

Is there anyway to obtain ds in GREIT algorithm in the same format to make plotting it consistent across algorithms?

**Comment by JeanRintoul** [comment]
Also, please let me know if this is your preferred mode of communication, or whether there is a forum or other for comments like this.

**Comment by liubenyuan** [comment]
For `greit` you can use `pcolorfast` from matplotlib,

```python
# suppose xg, yg are the grids along x- and y- axis, eit_im is the greit image
xv = xg[0]
yv = yg[:, 0]
eit_im = eit_im.reshape(xg.shape)
ax.pcolorfast(xv, yv, eit_im, cmap=your_cmap)
```

Sure, it's ok to communicate via the issues, currently, we do not have a forum yet.

**Comment by JeanRintoul** [comment]
Great - that works and should do the trick, although the different format still means different plotting methods needs to be used to get the same effect between algorithms, though perhaps that's unavoidable. ![figure_1](https://user-images.githubusercontent.com/315476/38692112-adf9ef7c-3e37-11e8-9b04-cf47e3a27f35.png)

**Comment by liubenyuan** [comment]
yes. In our group, GREIT has been used for pixel-based image analysis of EIT applications.
